utils/litemap/benches/bin/litemap_bincode.rs

```diff
@@ -43,7 +43,7 @@ const BINCODE: [u8; 278] = [
 /// Run this function to print new data to the console.
 #[allow(dead_code)]
 fn generate() {
-    let mut map = LiteMap::new();
+    let mut map: LiteMap<_, _, Vec<_>> = LiteMap::new();
```

Comment:
Question: should we consider making a new function like `LiteMap::new_with_store()` so that we can keep `LiteMap::new()` to imply the Vec-based version?

Reply:
I was thinking the other way around actually, `new` and `new_vec`

Reply:
I'm fine with that

Reply:
Change these to `new_vec` then